application/frontend/config/main.php

```diff
@@ -30,14 +33,18 @@
             // http://www.yiiframework.com/doc-2.0/guide-rest-routing.html
             'rules' => [
                 [
+                    // http://www.yiiframework.com/doc-2.0/yii-rest-urlrule.html
                     'class' => 'yii\rest\UrlRule',
-                    'controller' => ['user', 'authentication'],
+                    'controller' => ['authentication', 'user'],
+                    'extraPatterns' => [
```

Comment:
not needing those `405`'s simplifies things greatly.  https://github.com/silinternational/idp-id-broker/commit/af1e2cd61eb2acb345276fdd9fad4552c87f7524